Doppler.BigQueryMicroservice/Controllers/BigQueryController.cs

```diff
@@ -13,19 +13,31 @@ namespace Doppler.BigQueryMicroservice.Controllers
     [ApiController]
     public class BigQueryController : ControllerBase
     {
-        private readonly IUnitOfWork unitOfWork;
+        private readonly IUnitOfWork _unitOfWork;
         public BigQueryController(IUnitOfWork unitOfWork)
         {
-            this.unitOfWork = unitOfWork;
+            this._unitOfWork = unitOfWork;
         }
 
         [HttpGet("/big-query/{accountName}/allow-emails")]
```

Comment:
Está fuera del scope de este PR, pero en mi opinión esto debería ser `allowed` emails (en español sería _"permitido"_ en lugar de _"permitir"_)

```suggestion
        [HttpGet("/big-query/{accountName}/allowed-emails")]
```